app/src/main/java/org/aplatanao/billing/cockpit/models/Log.java

```diff
@@ -0,0 +1,40 @@
+package org.aplatanao.billing.cockpit.models;
+
+import javafx.beans.property.SimpleStringProperty;
+
+public class Log {
+    private final SimpleStringProperty dateCol;
```

Comment:
do not use a suffix for property names, since the entry does not need to know anything about a table or possible other bindings